src/App/Model.elm

```diff
@@ -80,19 +80,18 @@ type alias Flags =
 
 
 type alias Model =
-    { currentPage : Page
-    , flags : Flags
-    , pages :
-        { home :
-            { promotedArticles : WebData (List Article)
-            , promotedRecipes : WebData (List Recipe)
-            , recipes : WebData (List Recipe)
-            }
-        , articles : WebData (List Article)
-        , recipes : Dict String (WebData (List Recipe))
-        , recipe : { recipe : WebData Recipe, recipes : WebData (List Recipe) }
-        , contact : ContactForm
+    { flags : Flags
+    , currentPage : Page
+    , pageHome :
+        { promotedArticles : WebData (List Article)
+        , promotedRecipes : WebData (List Recipe)
+        , recipes : WebData (List Recipe)
         }
+    , pageAboutUs : {}
+    , pageRecipes : Dict String (WebData (List Recipe))
```

Comment:
Dict doesn't maintain order -- don't you prefer to DictList this?

Also, is the `String` the entity id or uuid? In that case I'd do something like (a bit fancy example, but we use in real projects):

```elm

type RecipeId = RecipeId String

-- Or if you wanted to keep also the UUID and ID
type RecipeIdAndUuid = RecipeId String Int

type alias RecipeDictList = EveryDictList RecipeId (WebData Recipe)
```